### ISSUE TITLE
Falcon Client: Cannot find module 'lodash'

### ISSUE THREAD

**Issue by prosellen** [deity-io/falcon#311]
**Describe the bug**
After creating a new app, the client exits with the uncaught exception "Cannot find module 'lodash'"

**To Reproduce**
1. Create new app with `yarn create falcon-app falcon`
2. `cd C:\Users\<user>\projects\falcon\client`
3. `yarn start`
4. See error:
```yarn start
yarn run v1.13.0
$ falcon-client start
Uncaught Exception:  { Error: Cannot find module 'lodash'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (C:\Users\peter.rosellen\projects\deity-falcon\client\node_modules\http-proxy-middleware\lib\index.js:1:84)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3) code: 'MODULE_NOT_FOUND' }
Done in 1.25s.```

**Expected behavior**
Server should start

**Desktop:**
 - OS: Windows 10
 - Node: v10.15.1
 - Yarn: 1.13.0
 - create-falcon-app: 1.1.5

**Comment by prosellen** [comment]
Removing `node_modules` and re-installing with `npm i` solves this.

However, removing `node_modules` and re-installing with `yarn install`, produces the same error.

**Comment by cane-lagerwaard** [comment]
@prosellen this issue should be fixed in the latest release `1.0-rc`, can you confirm?
